pkg/ops/iso.go

```diff
@@ -31,10 +38,42 @@ func GenISO(name, src, dst string, i schema.ISO) func(ctx context.Context) error
 		}
 
 		log.Info().Msgf("Generating iso '%s' from '%s' to '%s'", name, src, dst)
-		out, err := utils.SH(fmt.Sprintf("/entrypoint.sh --debug --name %s build-iso --squash-no-compression --overlay-iso %s --date=false --output %s dir:%s", name, overlay, dst, src))
-		log.Printf("Output '%s'", out)
+		cfg, err := enkiconfig.ReadConfigBuild("/config", &pflag.FlagSet{})
+		if err != nil {
+			return err
+		}
+		cfg.Name = name
+		cfg.OutDir = dst
+		cfg.Logger = sdkTypes.NewKairosLogger("enki", "debug", false)
+		imgSource, err := v1.NewSrcFromURI("dir:" + src)
```

Comment:
why not `v1.NewDirSrc(src)` ? we expect the source to be a valid dir already no? If it needs validation it would be on the CLI side, not here?

Reply:
Done, changed to NewDirSrc